src/components/__tests__/Book-test.tsx

```diff
@@ -18,7 +18,7 @@ let book: BookData = {
 };
 
 describe("Book", () => {
-  it("shows the book cover", () => {
+  it("shows the book cover with alt", () => {
```

Comment:
we decided to remove the alt, but i forgot to remove it from the test description